addons/mail/models/mail_thread.py

```diff
@@ -3861,6 +3861,29 @@ def _notify_by_web_push_prepare_payload(self, message, msg_vals=False):
         else:
             icon = '/web/static/img/odoo-icon-192x192.png'
 
+        if tools.is_html_empty(body) and message.attachment_ids:
+            total_attachments = len(message.attachment_ids)
+            # sudo: ir.attachment - access voice_ids linked to an attachment, if present.
+            attachments = message.attachment_ids.sudo()
+
+            def get_attachment_label(attachment):
+                return self.env._("Voice Message") if attachment.voice_ids else attachment.name
+
+            if total_attachments == 1:
+                body = get_attachment_label(attachments[0])
+            elif total_attachments == 2:
+                body = self.env._(
+                    "%(attachment_one)s and %(attachment_two)s",
+                    attachment_one=get_attachment_label(attachments[0]),
+                    attachment_two=get_attachment_label(attachments[1]),
+                )
+            else:
+                body = self.env._(
+                    "%(attachment_name)s and %(count)d other attachments",
+                    attachment_name=get_attachment_label(attachments[0]),
+                    count=total_attachments - 1,
+                )
```

Comment:
would use `file1 and file2` string like in JS, again it's simpler for translator and also this will be reconciled to a single translation entry if they are the same string, thus half the work for translator ;)